### ISSUE TITLE
BUG Eroding Salt Mulch

### ISSUE THREAD

**Issue by 19Luzifer89** [benbenlaw/seaopolis#156]
Hey, I have the following problem with the quest with this Eroding Salt Mulch because if I make 4 Sandy mulch and a Normal Mulch block in the middle and click on it with this Eroding Salt Mulch, unfortunately nothing happens, which means I can't get any further in the quest because I can't get to the liquid

**Comment by moody69420** [comment]
so u placed mulch and right clicked it with eroding salt mulch? I was just able to make the eroding fluid myself.

**Comment by 19Luzifer89** [comment]
Yes, I did but nothing happens :(

**Comment by Azulear** [comment]
You can try again with the salt in your offhand.

**Comment by 19Luzifer89** [comment]
Okay, it worked now in the offhand but in the one YouTube video he has it in the mainhand

**Comment by Azulear** [comment]
Normally it should work in the main hand, but sometimes it doesn't and you have to reload the game but using it in the offhand is a faster way around for it.

**Comment by 19Luzifer89** [comment]
Yes, thank you for the quick support and I hope for many cool further updates for the modpack, it's really a lot of fun :)

**Comment by andreysantos** [comment]
mine is having the same problem, I click on it with Eroding Salt Mulch and nothing happens to the mulch

**Comment by CharZinta** [comment]
> mine is having the same problem, I click on it with Eroding Salt Mulch and nothing happens to the mulch

Please create a new thread as your bug is not the one mentioned before. This one is for "main hand not always working" Vs not working at all.

Please post, in a new thread,  what version of the pack and if your paying single player or server.

**Comment by andreysantos** [comment]
> > o meu está tendendo o mesmo problema, eu clico nele com Eroding Salt Mulch e nada acontece com a cobertura morte
> 
> Por favor, crie um novo tópico, pois seu bug não é o mencionado antes. Este é para "mão principal nem sempre funciona" Vs não funciona em tudo.
> 
> Por favor, publique, em um novo tópico, qual versão do pacote e se você paga single player ou servidor.

Thank you, following the tips, I'll be able to solve it, I'm using the new version
